creator-node/package.json

```diff
@@ -12,8 +12,11 @@
     "test:unit": "./scripts/run-tests.sh unit_test",
     "coverage": "nyc npm run test",
     "report": "nyc report --reporter=html",
-    "lint": "./node_modules/.bin/standard",
-    "lint-fix": "./node_modules/.bin/standard --fix"
+    "lint": "npm run prettier:check",
+    "lint:fix": "npm run prettier:write",
+    "prettier:base": "./node_modules/.bin/prettier --parser typescript --single-quote --no-semi --trailing-comma none",
+    "prettier:check": "npm run prettier:base -- --list-different \"src/**/*.{js,ts}\"",
```

Comment:
do we also need to add things in `scripts/` and `test/` dirs?

Reply:
i wanted to keep this tightly scoped by starting with just the code first. you think it's worth doing scripts and test as well now?

Reply:
think minimal scope makes sense